Objects/unicodeobject.c

```diff
@@ -206,6 +206,11 @@ extern "C" {
             *_to++ = (to_type) *_iter++;                \
     } while (0)
 
+#define LATIN1(ch)  \
+    (ch < 128 ? \
+     (PyObject*)&_Py_SINGLETON(strings).ascii[ch] \
+     : (PyObject*)&_Py_SINGLETON(strings).latin1[ch - 128])
```

Comment:
FWIW, I prefer to put the "?" at the front of the line (like you did with ":") when splitting a ternary expression across lines:

```suggestion
    (ch < 128 \
     ? (PyObject*)&_Py_SINGLETON(strings).ascii[ch] \
     : (PyObject*)&_Py_SINGLETON(strings).latin1[ch - 128])
```

Reply:
Done